appnexus/lucid
# Cxp 805 validation state to search select fields

## PR Checklist

- Manually tested across supported browsers
  - [ ] Edge (Win 10)

// please check out SearchableMultiSelect, SearchableSingleSelect, SearchableSelect
https://docspot.adnxs.net/projects/lucid/CXP-805-validation-state-to-search-select-fields/?path=/docs/controls-searchablemultiselect--invalid
